mj_envs/hand_manipulation_suite/relocate_v0.py

```diff
@@ -13,10 +15,10 @@ def __init__(self):
         mujoco_env.MujocoEnv.__init__(self, curr_dir+'/assets/DAPG_relocate.xml', 5)
         
         # change actuator sensitivity
-        self.sim.model.actuator_gainprm[self.sim.model.actuator_name2id('A_WRJ1'):self.sim.model.actuator_name2id('A_WRJ0')+1,:] = np.array([10, 0, 0])
-        self.sim.model.actuator_gainprm[self.sim.model.actuator_name2id('A_FFJ3'):self.sim.model.actuator_name2id('A_THJ0')+1,:] = np.array([1, 0, 0])
-        self.sim.model.actuator_biasprm[self.sim.model.actuator_name2id('A_WRJ1'):self.sim.model.actuator_name2id('A_WRJ0')+1,:] = np.array([0, -10, 0])
-        self.sim.model.actuator_biasprm[self.sim.model.actuator_name2id('A_FFJ3'):self.sim.model.actuator_name2id('A_THJ0')+1,:] = np.array([0, -1, 0])
+        self.sim.model.actuator_gainprm[self.sim.model.actuator_name2id('A_WRJ1'):self.sim.model.actuator_name2id('A_WRJ0')+1,:3] = np.array([10, 0, 0])
+        self.sim.model.actuator_gainprm[self.sim.model.actuator_name2id('A_FFJ3'):self.sim.model.actuator_name2id('A_THJ0')+1,:3] = np.array([1, 0, 0])
```

Comment:
It's a little bizarre that we need to relax the actuator only for FF knuckle. Either this indicates that we should do it for all knuckles or its a typo. I'm inclined to believe that this is the later -- (a) policies are less likely to be sensitive to knuckles actuator strength (b) Arm (ARJ3) is more likely to have increased sensitivity.

Reply:
Is this related to the original version or my updated version? The only change from the original is that in mujoco 1.5, there are 3 columns in the actuator fields, whereas in mujoco 2.0 there are 10 fields. Thus the columns are modified from `:` to `:3`. The names of actuators and values themselves are unchanged in this PR.

Reply:
This is related to the original version, to begin with. My hypothesis is that either we don't need to update A_FFJ3 at all, or its a typo for A_ARJ3

Reply:
I see, can you try both versions and let me know? The train script in hand_dapg should be super handy for this. You can just run it with the two versions, and we can finalize after that. In any case, the version in the PR gives good results and movements.